admin/scripts/modules/uvod.php

```diff
@@ -98,7 +98,15 @@
 }
 
 if(post('zmenitUdaj')) {
-  dbUpdate('uzivatele_hodnoty', post('udaj'), ['id_uzivatele' => $uPracovni->id()]);
+  if(post('udaj')['login_uzivatele']) {
+    try {
+      dbUpdate('uzivatele_hodnoty', post('udaj'), ['id_uzivatele' => $uPracovni->id()]);
+    } catch (Exception $e) {
+      throw new Chyba('<strong>Jejda!</strong> Uživatel se stejnou přezdívkou již existuje.');
+    }
+  } else {
+    dbUpdate('uzivatele_hodnoty', post('udaj'), ['id_uzivatele' => $uPracovni->id()]);
+  }
```

Comment:
Jak správně píšeš v commentu, takto to udělat nelze. Při odchytávání generické výjimky netušíš co se děje (např.: selhání připojení k databázi vypíše "uživatel už existuje"). I prodloužení kódu na devítinásobek považuji za problém. Je možné, že update vydá DbDuplicate exception nebo podobně, s tím by se pracovat už dalo.